include/wrapper.sh

```diff
@@ -18,7 +22,51 @@ HTTP_PROXY=""
 : ${HTTP_PROXY_PORT:=""}
 : ${HTTP_PROXY_USERNAME:=""}
 : ${HTTP_PROXY_PASSWORD:=""}
+: ${ZK_JAVA_INITIAL_HEAP_SIZE:=$DEFAULT_ZK_JAVA_INITIAL_HEAP_SIZE}
```

Comment:
Can you remind me, what this variable instantiation comes from?
`: ${SOMETHING:=$SOMETHING_ELSE}` not bash, right?